### DISCUSSION TITLE
版本升级

### DISCUSSION THREAD

**Discussion by young138120** [apache/doris#16178] [A - General / Q&A]
近期服务器会有一次重启  
想要借东风进行一次版本的升级，预期冲1.0.0-rc03升级到1.2.1版本  
看官网介绍不能跨两个版本升级，然后fe需要是高可用 这两个不满足  
我是一个follow-fe，一个observer ，三个be

能否直接暴力听服务直接往上升？

**Comment by zy-kkk** [comment]


**Reply by young138120** [reply]
测试兼容性 主要是指的哪方面呢？
fe和be启动的日志情况吗？
我再测试环境把除了meta数据，其他都替换成1.2.1
目前再观察日志，测试环境没有什么数据

**Reply by zy-kkk** [reply]
[BE升级正确性](https://doris.apache.org/zh-CN/docs/dev/admin-manual/cluster-management/upgrade#%E6%B5%8B%E8%AF%95-be-%E5%8D%87%E7%BA%A7%E6%AD%A3%E7%A1%AE%E6%80%A7)
[FE元数据兼容性](https://doris.apache.org/zh-CN/docs/dev/admin-manual/cluster-management/upgrade#%E6%B5%8B%E8%AF%95-fe-%E5%85%83%E6%95%B0%E6%8D%AE%E5%85%BC%E5%AE%B9%E6%80%A7)
